playground/src/App.js

```diff
@@ -21,6 +21,7 @@ function App() {
           anonymousMode={true}
           headerColor="white"
           toastBarPosition="bottom-end"
+          showRoles={true}
```

Comment:
please update this change in the README `props`

Reply:
resolved in [52ebd6b](https://github.com/RocketChat/EmbeddedChat/pull/115/commits/52ebd6bd133397d706cfb7139afc8bac43070fb5)